src/pages/signin/SignInPage.js

```diff
@@ -81,21 +81,18 @@ class SignInPage extends Component {
         const welcomeText = this.props.translate(`welcomeText.${showPasswordForm ? 'phrase4' : 'phrase1'}`);
 
         return (
-            <>
-                <SafeAreaView style={[styles.signInPage]}>
-                    <SignInPageLayout
-                        welcomeText={welcomeText}
-                        shouldShowWelcomeText={showLoginForm || showPasswordForm || !showResendValidationLinkForm}
-                        shouldShowWelcomeScreenshot={showLoginForm}
-                    >
-                        {showLoginForm && <LoginForm />}
-
-                        {showPasswordForm && <PasswordForm />}
-
-                        {showResendValidationLinkForm && <ResendValidationForm />}
-                    </SignInPageLayout>
-                </SafeAreaView>
-            </>
+            <SafeAreaView style={[styles.signInPage]}>
+                <SignInPageLayout
+                    welcomeText={welcomeText}
+                    shouldShowWelcomeText={showLoginForm || showPasswordForm || !showResendValidationLinkForm}
+                    shouldShowWelcomeScreenshot={showLoginForm}
+                >
+                    {/* Both LoginForm and PasswordForm should be present in the UI for Password Managers to work. */}
```

Comment:
I think we need to say more. Can we please explain that the fields remain accessible if hidden but not if unmounted? Basically this should read as a warning that password managers will break without using `isVisible` which "hides" but does not "unmount". We don't want anyone to break this unintentionally.

Reply:
I'll try my best but really bad at explaining.

Reply:
No worries, something like this works:

>LoginForm and PasswordForm must use the isVisible prop. This keeps them mounted, but visually hidden so that password managers can access the values. Conditionally rendering these components will break this feature.